extras/fraxtal.json

```diff
@@ -3,6 +3,9 @@
         "zero": "0x0000000000000000000000000000000000000000"
     },
     "tokens": {},
+    "frax": {
+        "incentivizer_amo_v3_2": "0xe9b3f9cd8851216fBb828da590CCA8CE7F03E863"
```

Comment:
Note that per Nader AMOs are per coin.  So this is the SFRAX AMO, there will likely be other AMOs for other frax tokens. 
```suggestion
        "sfrax_amo_v3_2": "0xe9b3f9cd8851216fBb828da590CCA8CE7F03E863"
```

Reply:
ah you are right. well half. it is for fxs actually:

<img width="364" alt="Screenshot 2024-09-03 at 13 30 01" src="https://github.com/user-attachments/assets/f9d2fffd-8fae-4c04-bc63-496eae2d94cd">

Reply:
I think all AMOs distribute FXS (and maybe other tokens) but are deployed per base pair token.  I can add you to the chat with Nader if you want to ask more about it.